src/styles/footer.scss

```diff
@@ -0,0 +1,23 @@
+.footer {
+  position: fixed;
+  bottom: 0;
+  z-index: -1;
+  width: 100%;
+  height: $footerHeight;
+  background: $buttonHoveredShadow url("../images/footer.jpg") center;
+  background-size: cover;
+  @media (max-width: $displayLaptop) {
+    height: $footerHeightMob;
```

Comment:
this variable can be removed as well

Reply:
there are 3 variables. What kind?

Reply:
comment is usually for the last line if other is not specified under the code block

Reply:
So I think no. Because height of footer and margin-bottom of main are in different files.